scss/components/tile.scss

```diff
@@ -75,9 +75,13 @@ $block: #{$fd-namespace}-tile;
   &__title {
     @include fd-type("0");
     @include fd-var-color("color", $fd-tile-title-color, --fd-tile-title-color);
-    margin-bottom: 0;
+    @include fd-reset-spacing();
   }
   &__actions {
     padding: fd-space(3) $fd-tile-content-padding-x fd-space(3) fd-space(3);
   }
+
+  .fd-has-background-color-accent-3 {
+    background-color: var(--fd-color-accent-3, map-get($fd-colors-accent, 3));
+  }
```

Comment:
This class has nothing to do with the tile component.  It should either be renamed to be part of tile or moved to a shared location.  See #157 for a similar comment.